src/pages/settings/index.tsx

```diff
@@ -1,3 +1,26 @@
-import SetupPage from './setup'
+import { useEffect } from 'react'
+import type { NextPage } from 'next'
+import Head from 'next/head'
+import { useRouter } from 'next/router'
+import { generalSettingsNavItems, settingsNavItems } from '@/components/sidebar/SidebarNavigation/config'
 
-export default SetupPage
+const Settings: NextPage = () => {
+  const router = useRouter()
+
+  useEffect(() => {
+    const redirectPath = router.query.safe ? settingsNavItems[0].href : generalSettingsNavItems[0].href
```

Comment:
When I am on the safe settings page and I click on "Settings" in the Footer, the general settings page opens and takes me out of my safe. I would suggest redirecting to `settingsNavItems[0].href` in that case as well so it doesn't navigate away from the current safe.

Reply:
If you want to go to the Safe settings, you must click on "Settings" in the Safe nav sidebar.

I could rename Settings in the footer to Preferences to avoid confusion?